### DISCUSSION TITLE
Smoothened Pole figure with MUD and IPF from Euler angle data

### DISCUSSION THREAD

**Discussion by makeabhishek** [mtex-toolbox/mtex#1011] [Ask Anything] [answered]
Hi, I want to plot pole figure from the Euler angle data which is obtained for a Titanium-64 alloy with hexagonal and cubic phases. 

**1. How to make a polar plot of both cubic and hexagonal crystal with MUD value as shown below figure? How to see the legend of the pole figure and IPF?
2. How to plot IPF (IPF-Z, IPFX, IPFY) for the same?
3. How to plot ODF from Euler angle data?**

![image](https://user-images.githubusercontent.com/47937684/117273187-78310680-ae8e-11eb-8cdd-3cd69b6007d6.png)

**Comment by kilir** [comment, marked as answer]
Hi,
this is a good start https://mtex-toolbox.github.io/Documentation.html where all this is explained.
For example plotting inverse polefigures https://mtex-toolbox.github.io/OrientationInversePoleFigure.html or various ways of plotting ODF data https://mtex-toolbox.github.io/ODFPlot.html,  https://mtex-toolbox.github.io/EulerAngleSections.html or https://mtex-toolbox.github.io/SigmaSections.html and so on.
In case anything remains unclear, please ask here.

Cheers,
Rüdiger
